internal/command/darwin_image.go

```diff
@@ -31,6 +32,7 @@ func (cmd *DarwinImage) Description() string {
 // Parse parses the arguments and set the usage for the command
 func (cmd *DarwinImage) Parse(args []string) error {
 	flagSet.StringVar(&cmd.sdkPath, "xcode-path", "", "Path to the Command Line Tools for Xcode (i.e. /tmp/Command_Line_Tools_for_Xcode_12.4.dmg")
+	flagSet.StringVar(&cmd.sdkVersion, "sdk-version", "11.1", "SDK Version to use")
```

Comment:
This may be the source of @Jacalz problem - if we default to no-specified version then it should work.
The problem with defaulting to this version is that it may not be included in the currently-downloaded SDK bundles I think.

Reply:
This is a good improvement thanks.
I wonder, should we default to the newest version instead of failing if multiple are found? (That is closer to what macOS tooling would do).

Reply:
Good point. Updated in e1a0395

Reply:
I think the update to 11.3 broke the fallback to whatever is installed if 11.3 is missing.
If I only have 11.2 installed now it won't compile.

Reply:
> By "should we default to the newest version instead" I had meant to ask if we should check what the newest installed SDK is and compile against that if not specified.

Do you mean build the docker image using the latest SDK available in the XCode Command Line Tool  package?
If so, I agree. This should be the best UX, but it requires some effort that probably does not worth the benefit.

The image need to be built once in while, supporting out of the box a "suggested/supported" XCode version with a relative SDK should good enough for now IMO.

At writing time the suggested version of XCode Command Line Tools version should be v12.5 targeting SDK 11.3.

Reply:
@Jacalz don't get me wrong, I agree too :-) 
What I'm saying is that I'd move this into a separated task to not block the release since it is not an easy pick and requires some effort to be implemented ;-)

Reply:
I agree also that this is too much work and should not hold up the release.
However I have a concern that by specifying a version here we run the risk of breaking what could be the simple/common workflow...

The question is - do we want to break it for all people building the macOS image, or just those who have multiple SDKs installed.
I suspect that there is a significant overlap between the two as most will have many. However new developers will likely just have one (having just downloaded the tools?) and if possible it would be good to keep that as smooth as possible.

Reply:
As per our chat in Slack, the code has been updated to work as below:
- default mode to auto-detect. If no sdk-version is provided it will try to detect and use the newest SDK present in the Xcode package
- The SDK can be specified via the `sdk-version` flag. If that SDK is not provided by the Xcode package the build image task will fail.